people-and-planet-ai/image-classification/README.ipynb

```diff
@@ -0,0 +1,808 @@
+        "\n",
+        "This notebook code lets you train and deploy an ML model, as well as test its accuracy, from end-to-end. When you run a code cell, the code runs in the notebook's runtime, so you're not making any changes to your personal computer.\n",
+        "\n",
+        "> 🛎️ **To avoid any errors**, wait for each section to finish in their order before clicking the next “run” icon.\n",
+        "\n",
+        "This sample must be connected to a Google Cloud project, but nothing else is needed other than your Google Cloud project.\n",
+        "You can use an existing project and the cost will be less than \\$30. Alternatively, you can create a new Cloud project with cloud credits for free"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "Kb9doxCpUcpF"
+      },
+      "source": [
+        "## 🚴‍♀️ Steps included\n",
+        "\n",
+        "The notebook leverages camera trap images dataset from the\n",
+        "[WCS Camera Traps dataset](http://lila.science/datasets/wcscameratraps)\n",
+        "from the _Labeled Information Library of Alexandria: Biology and Conservation_ (LILA) organization.\n",
+        " \n",
+        "Here's a quick summary of what you’ll go through:\n",
+        "\n",
+        "1. **Create a database of image metadata** _(~5 minutes to complete, costs a few cents)_:\n",
+        "   in a\n",
+        "   [BigQuery](https://cloud.google.com/bigquery)\n",
+        "   table with all the image file names along with their respective category. \n",
+        "\n",
+        "1. **Train an image classifier** _(~2 hours to complete, costs ~$25.00)_:\n",
+        "   With some desired criteria, a\n",
+        "   [Dataflow](https://cloud.google.com/dataflow)\n",
+        "   pipeline creates a balanced dataset from the metadata database.\n",
+        "   It then physically downloads the necessary images from LILA into\n",
+        "   [Cloud Storage](https://cloud.google.com/storage),\n",
+        "   imports the data into\n",
+        "   [AI Platform Unified](https://cloud.google.com/ai-platform-unified/docs/start/introduction-unified-platform),\n",
+        "   and starts the model training job.\n",
+        "\n",
+        "1. **Deploy an ML model** _(costs **$1.25 for every hour** the model is deployed online… so complete the “delete model” step soon)_:\n",
+        "   After the model finishes training, we look at the results and deploy it into a Cloud endpoint in AI Platform Unified.\n",
+        "\n",
+        "1. **Classify images with the model**:\n",
+        "   We send some images to the model and get back it’s classification predictions.\n",
+        "\n",
+        "1. **Delete the project** (optional) to avoid ongoing costs."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "VRZjMwyf2BZE"
+      },
+      "source": [
+        "## ✨ Before you begin, you need to…\n",
```

Comment:
I think the next few sections belong under this headers e.g.

## Before you begin
...
### Entering project details
### Preparing project environment 
### Creating the database ...

Reply:
Thanks, I left the "Creating the database" section as its own since that is technically now running the sample code, but I moved the other two inside "Before you begin".